src/main/java/hudson/plugins/timestamper/TimestampNote.java

```diff
@@ -104,4 +113,11 @@ public Timestamp getTimestamp(Run<?, ?> build) {
     }
     return null; // each time-stamp note affects one line only
   }
+
+  public void plain(OutputStream out) throws IOException {
+    TimestamperConfig config = GlobalConfiguration.all().get(TimestamperConfig.class);

```

Comment:
Can this not be retrieved once (statically) and avoid the lookup for every log? All that stuff is static anyway.